apps/cli/src/bin.ts

```diff
@@ -108,7 +111,7 @@ program
           if (path.endsWith(".typegen.ts")) {
             return;
           }
-          tasks.push(writeToFiles([path]));
+          tasks.push(writeToFiles([path], opts.silent));
```

Comment:
Might be more scalable to just pass `opts` in here? That way we can avoid multiple arguments in the future.